samples/menus/nav-drawer/add-mini/index.html

```diff
@@ -11,31 +11,33 @@
   </head>
 
   <body>
-    <div id="root" style="display: flex;">
-        <igc-nav-drawer id="navDrawer">
-            <igc-nav-drawer-item>
-                <igc-icon slot="icon" name="home"></igc-icon>
-                <span slot="content">Home</span>
-            </igc-nav-drawer-item>
-
-            <igc-nav-drawer-item>
-                <igc-icon slot="icon" name="search"></igc-icon>
-                <span slot="content">Search</span>
-            </igc-nav-drawer-item>
-
-            <div slot="mini">
+    <div id="root">
```

Comment:
This sample is throwing error:
Uncaught TypeError: Cannot set properties of null (setting 'onclick')
    at HTMLElement.connectedCallback (index.ts:73)
    at Router.displaySample (router.ts:53)
    at Router.navigateToRoute (router.ts:108)
    at async Router.connect (router.ts:34)